freqtrade/rpc/rpc.py

```diff
@@ -763,21 +763,22 @@ def _rpc_reload_trade_from_exchange(self, trade_id: int) -> Dict[str, str]:
 
     def __exec_force_exit(self, trade: Trade, ordertype: Optional[str],
                           amount: Optional[float] = None) -> bool:
-        # Check if there is there is an open order
+        # Check if there is there is open orders
         fully_canceled = False
-        if trade.open_order_id:
-            order = self._freqtrade.exchange.fetch_order(trade.open_order_id, trade.pair)
+        for oo in trade.open_orders:
+            order = self._freqtrade.exchange.fetch_order(oo.order_id, trade.pair)
 
             if order['side'] == trade.entry_side:
                 fully_canceled = self._freqtrade.handle_cancel_enter(
-                    trade, order, CANCEL_REASON['FORCE_EXIT'])
+                    trade, order, oo.order_id, CANCEL_REASON['FORCE_EXIT'])
```

Comment:
you'll need to be careful here
changing if with for will change the context - and `fully_canceled` will have the result of the last order - which may or may not be correct behavior

Reply:
True, I changed that